### DISCUSSION TITLE
Error submitting job

### DISCUSSION THREAD

**Discussion by minminfu** [E3SM-Project/E3SM#6241] [E3SM model help]
Hi, I am a new E3Sm user. Every time I try to submit a case I have built, I receive this error:

ERROR: Command: '/global/cfs/cdirs/m4340/E3SMv2_1/components/eam/bld/build-namelist -infile /pscratch/sd/m/mjfu/E3SM/e3sm_generic_WCYCL1850_v1/tests/XS_2x5_ndays/case_scripts_0501/Buildconf/eamconf/cesm_namelist -csmdata /global/cfs/cdirs/e3sm/inputdata -ignore_ic_date -use_case 1850_eam_CMIP6 -inputdata /pscratch/sd/m/mjfu/E3SM/e3sm_generic_WCYCL1850_v1/tests/XS_2x5_ndays/case_scripts_0501/Buildconf/eam.input_data_list -ntasks 320.0 -namelist " &atmexp  /" ' failed with error 'ERROR(Build::Namelist::_parse_next): expect a F90 constant for a namelist instead got: /' from dir '/pscratch/sd/m/mjfu/E3SM/e3sm_generic_WCYCL1850_v1/tests/XS_2x5_ndays/case_scripts_0501/Buildconf/eamconf'

Does anybody have any suggestions for how to resolve this? I am using the WCYCL1850 compset and ne30pg2_EC30to60E2r2 resolution on Perlmutter, with E3SM version 2.1

**Comment by rljacob** [comment]
What is the full create_newcase command?

**Reply by minminfu** [reply]
/global/cfs/projectdirs/m4340/E3SMv2_1/cime/scripts/create_newcase \
                        --case e3sm_generic_WCYCL1850_v1\
                        --output-root /pscratch/sd/m/mjfu/E3SM/e3sm_generic_WCYCL1850_v1\
                        --script-root /pscratch/sd/m/mjfu/E3SM/e3sm_generic_WCYCL1850_v1/tests/XS_2x5_ndays/case_scripts_0501\
                        --handle-preexisting-dirs u \
                        --compset WCYCL1850 \
                        --res ne30pg2_EC30to60E2r2 \
                        --machine pm-cpu \
                        --project m4340 \
                        --walltime "00:30:00" \
                        --pecount XS
